content/posts/02_staff_expectations/index.md

```diff
@@ -0,0 +1,116 @@
+---
+title: "What is expected from staff engineers"
+date: 2022-12-18
+tags: [
+    "career", "growth"
+]
+---
+
+It's relatively easy to explain expectations for junior, mid, and even senior engineers. But it starts to get much harder at staff+ 
+positions. Career progression broadens the scope of work andresponsibilities. The environment and problems get more ambiguous. It's not easy
+to define a clear and measurable set of expectations, because staff roles also differ between companies and
+their actual needs. They can also vary between different [staff engineer archetypes](https://staffeng.com/guides/staff-archetypes).
+
+In the article, we will focus mostly on the **Tech Lead** archetype, which is the most popular in the industry. We will present some 
```

Comment:
`which is the most popular in the industry` do we have anything to prove it or is it our gut feeling only?

Reply:
I doubt there are numeric data to back up this opinion. But Will Larson is writing about this https://staffeng.com/guides/staff-archetypes/ and many major companies (like FANG) have tech lead roles.